botorch_community/posteriors/bll_posterior.py

```diff
@@ -9,31 +9,31 @@
 import math
 
 import torch
-from botorch.posteriors import GPyTorchPosterior, Posterior
-
+from botorch.posteriors import GPyTorchPosterior
 from botorch_community.models.blls import AbstractBLLModel
+from gpytorch.distributions import MultivariateNormal
 
 from torch import Tensor
 
 
-class BLLPosterior(Posterior):
+class BLLPosterior(GPyTorchPosterior):
     def __init__(
         self,
-        posterior: GPyTorchPosterior,
         model: AbstractBLLModel,
+        mvn_dist: MultivariateNormal,
```

Comment:
`GPyTorchPosterior` uses `distribution` for this arg: https://github.com/pytorch/botorch/blob/41d0645f9e8d7e1d24396fa322ec8a6b8469fac9/botorch/posteriors/gpytorch.py#L40, let's be consistent with that?